contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -118,7 +130,7 @@ script: |
   # Extract the git archive into a dir for each host and build
   for i in ${HOSTS}; do
     export PATH=${BASEPREFIX}/${i}/native/bin:${ORIGPATH}
-    if [ "${i}" = "riscv64-linux-gnu" ]; then
+    if [ "${i}" = "riscv64-linux-gnu" ] || [ "${i}" = "powerpc64-linux-gnu" ] || [ "${i}" = "powerpc64le-linux-gnu" ]; then
```

Comment:
Why does this remain conditional? We enforce it on all platforms; might as well make it explicit.

Reply:
Because that is unsafe, see rationale here: https://github.com/bitcoin/bitcoin/pull/20963#discussion_r560919190

These lines can go away completely after gcc upgrade that fixes the underlying issue. But no sooner.